microsoft/PowerToys
# Sync changes from master (dev/imageResizer)

## Summary of the Pull Request
This PR pulls the latest changes from master to the dev/imageResizer branch.


## PR Checklist
* [X] Applies to #53 


## Validation Steps Performed
- Built release x64 and installed with MSIX to verify all PowerToys were working. Also ran tests.

**Note:** After installing with MSIX explorer.exe will have to be restarted from Task Manager or you have to reboot/sign out for the shell extensions to get loaded. This is because of the change to the MSIX installer in master and this is tracked in #1326
